src/Couchbase.Lite/Couchbase.Lite.csproj

```diff
@@ -52,6 +52,7 @@
     <DefineConstants>$(DefineConstants);NEEDS_LITECORE_LOAD</DefineConstants>
   </PropertyGroup>
   <ItemGroup>
+    <PackageReference Include="Microsoft.Maui.Dependencies" Version="6.0.486" />

```

Comment:
This is exactly what I said not to do.  This brings in Maui to every target, even non-Maui ones.

Reply:
Removed.